### DISCUSSION TITLE
DarkFluent theme not changing Window's title bar color

### DISCUSSION THREAD

**Discussion by ShrutiJaiswal1494** [AvaloniaUI/Avalonia#6594] [Q&A]
Hi, 
I am using the `FluentDark` theme in my application but that seems to work for the `Window Content` but not the `title bar` color. Is there a way to set it explicitly? Thanks in advance.

**Comment by maxkatz6** [comment]
No. Windows theme is independed as it is expected.
But you might be interested in custom window template, see https://github.com/AvaloniaUI/XamlControlsGallery/blob/master/XamlControlsGallery/FluentWindow.cs

**Reply by ShrutiJaiswal1494** [reply]
@maxkatz6  Sorry, I did not get it completely. Do I need to use this package to set title bar color and other kinds of stuff for windows? Also, is there any reference on how to do that? Thanks

**Reply by maxkatz6** [reply]
It's not a package, it's an example project

**Reply by ShrutiJaiswal1494** [reply]
@maxkatz6  I do not see a way to set the Title bar color in this custom template. Is it something not implemented yet?

**Reply by maxkatz6** [reply]
In this custom template you have custom window frame. You can set it's title bar as you want.

**Comment by maxkatz6** [comment]
Or if you want to keep native windows, you can call Win32 apis and cocoa apis, but it's not related to avalonia anymore at this point and you can find answers for that. Note that you can get pointer to the window using ```window.PlatformImpl.Handle.Handle```
